lib/cufft/libcufft.jl

```diff
@@ -26,6 +25,42 @@ end
     end
 end
 
+@cenum cudaXtCopyType_t::UInt32 begin
+    LIB_XT_COPY_HOST_TO_DEVICE = 0
+    LIB_XT_COPY_DEVICE_TO_HOST = 1
+    LIB_XT_COPY_DEVICE_TO_DEVICE = 2
+end
+
+const cudaLibXtCopyType = cudaXtCopyType_t
+
+@cenum libFormat_t::UInt32 begin
+    LIB_FORMAT_CUFFT = 0
+    LIB_FORMAT_UNDEFINED = 1
+end
+
+const libFormat = libFormat_t
+
+struct cudaXtDesc_t
+    version::Cint
+    nGPUs::Cint
+    GPUs::NTuple{64,Cint}
+    data::NTuple{64,Ptr{Cvoid}}
+    size::NTuple{64,Csize_t}
+    cudaXtState::Ptr{Cvoid}
+end
+
+const cudaXtDesc = cudaXtDesc_t
+
+struct cudaLibXtDesc_t
+    version::Cint
+    descriptor::Ptr{cudaXtDesc}
+    library::libFormat
+    subFormat::Cint
+    libDescriptor::Ptr{Cvoid}
+end
+
+const cudaLibXtDesc = cudaLibXtDesc_t
```

Comment:
These probably shouldn't live in the CUFFT wrappers, but I don't see any other uses right now.